vnext/Desktop.IntegrationTests/RNTesterIntegrationTests.cpp

```diff
@@ -88,6 +88,8 @@ TEST_CLASS (RNTesterIntegrationTests) {
     Assert::AreEqual(L"This is from console.error", result.Message.c_str());
   }
 
+  BEGIN_TEST_METHOD_ATTRIBUTE(XHRSample)
+  END_TEST_METHOD_ATTRIBUTE()
   TEST_METHOD(XHRSample) {
```

Comment:
I think we might have talked about this before, but XHRTest seems to start some async work then immediately pass. Given it was passing before for a fully stubbed module, I'm not sure how much validation we get from it.

```
  componentDidMount() {
    try {
      this._download();
    } catch (e) {
      console.error(e);
    }

    TestModule.markTestPassed(true);
  }
```

Reply:
Right now very little:
- Fail if the native module crashes.
- Catch any errors during the request send operation.

Now that we can use a complete and proven HTTP module, I'll rewrite this test so it does something relevant (before merging) :)

Reply:
If you're rewriting the test, could you do so in the new IT project instead of the desktop IT's? Should be the same coverage since they're testing the same module.